checker/lib/README.developers

```diff
@@ -0,0 +1,11 @@
+Please read the README file in this directory first, as it is the file
```

Comment:
I'm not sure I understand the point of adding a new README.developers file.  If the file exists, it should be included in the distribution to minimize the difference between the distribution and the repository.  Furthermore, I would think that the comment about where the source code came from is interesting mostly to developers, but it wasn't moved to this file.  I would just keep all the information in a single README file to reduce confusion and make it easier to find and update information.